interface/forms/group_attendance/new.php

```diff
@@ -0,0 +1,105 @@
+<?php
+include_once("../../globals.php");
+include_once("$srcdir/api.inc");
+include_once ("$srcdir/group.inc");
+include_once("statics.php");
+
+$returnurl = 'encounter_top.php';
+
+$participants = getParticipants($therapy_group);
+?>
+<html>
+
+<head>
+<?php html_header_show();?>
+<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/datatables.net-jqui-1-10-11/css/dataTables.jqueryui.min.css" type="text/css">
+<script src="<?php echo $GLOBALS['assets_static_relative'];?>/jquery-min-1-9-1/index.js"></script>
+<script src="<?php echo $GLOBALS['assets_static_relative'];?>/jquery-ui-1-10-4/ui/jquery.ui.core.js"></script>
+<script src="<?php echo $GLOBALS['assets_static_relative'];?>/datatables.net-1-10-11/js/jquery.dataTables.min.js"></script>
+<script src="<?php echo $GLOBALS['web_root'];?>/library/dialog.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot']; ?>/interface/forms/group_attendance/js/functions.js"></script>
```

Comment:
Good to get in the habit when bringing in custom javascript to incorporate the cache busting mechanism:
``````
<script src="<?php echo $GLOBALS['web_root'];?>/library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
<script type="text/javascript" src="<?php echo $GLOBALS['webroot']; ?>/interface/forms/group_attendance/js/functions.js?v=<?php echo $v_js_includes; ?>"></script>
``````
So whenever you modify this script in the future, you increment the $v_js_includes number in version.php and that will ensure a users local cache of the javascript gets overwritten with the new version.